blocks/editable/format-toolbar.js

```diff
@@ -151,17 +152,24 @@ class FormatToolbar extends Component {
 						className="editable-format-toolbar__link-modal"
 						style={ linkStyle }
 						onSubmit={ this.submitLink }>
+						<label
```

Comment:
@afercia I could easily plug those things in, they were well implemented. I thought the overhead of the instances ids might not worth the much more complex flow of having the labels separate.

So whatever you decide @afercia / @aduth , let me know and I'll let this ready for merging.